app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::ItemsController < Api::ApiController
+  respond_to :json
+
+  before_action :authenticate
+
+  def create
+
+    @list = List.find(params[:list_id])
+    @items = @list.items
+    @item = @list.items.build(item_params)
+    @item.list = @list
```

Comment:
I believe line 10 will be populating `@item.list` so line 11 wouldn't be needed.